product_webcam/product_webcam_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        
+        <record id="action_client_employees_menu" model="ir.actions.client">
+            <field name="name">Open Employees Menu</field>
```

Comment:
There are some copy&paste artifacts from `hr_webcam`.

A proper base module in `web_webcam`(?) for all the common webcam-related web components would probably be the best solution.